src/osmf/js/osmf_flash.js

```diff
@@ -280,6 +280,31 @@
     };
 
     /**
+     * Sets the closed captions on the video element.
+     * @public
+     * @method OoyalaFlashVideoWrapper#setClosedCaptions
+     * @param {string} language Selected language of captions
+     * @param {object} closedCaptions The captions object
+     * @param {object} params The parameters object
+     */
+      
+     this.setClosedCaptions = function(language,closedCaptions,params){
+       var parameters = {language:language, closedCaptions:closedCaptions, params:params};
+        this.callToFlash("setVideoClosedCaptions()" , parameters);
```

Comment:
minor - there is some inconsistent spacing here.  I suggest spaces between input parameters to the function and reducing one leading space on this line